### PR TITLE
TSE Review

### PR DESCRIPTION
**Pull request by jacobo12montes**
We added the Rspecs for the methods created on the board.rb:
#display_board
#update_board
#valid_move?
#winner?

### CODE REVIEW

**Comment by Anna-Myzukina** [comment]
## Hi :clap: 

### You did a great job so far! :+1: 
### Just a few changes before it can be approved:


## [REQUIRED]
- Please add descriptions to the README file
screen
- It would be great to have screenshots that show how the app works in the Readme file.


### [OPTIONAL]
- ["How to Write Good Commit Messages: A Practical Git Guide"](https://www.freecodecamp.org/news/writing-good-commit-messages-a-practical-guide/)
```
Write your commit message in the imperative and capitalize: "Fix linter" and not "fixed bug"
or "Fixes bug."  This convention matches up with commit messages generated
by commands like git merge and git revert.
```

### Kindly make the changes and submit a new code review request.
Thanks!


*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*
*Happy coding * <g-emoji class="g-emoji" alias="computer" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f4bb.png">computer</g-emoji>

File: README.md [hunk 1]
```diff
@@ -10,9 +10,7 @@ Additional description about the project and its features.
```
Comment (Anna-Myzukina) [inline README.md:13]: Here you could add a short description of this project.

**Comment by WinterCore** [comment]
## The Minskins Code Review 17/08/2020

Great project.
@simandebvu (The readme rubocop) says "Your readme sucks". He also said that @WinterCore sucks.

**Comment by jacobo12montes** [comment]
Hi @Anna-Myzukina we have made the changes  requested
-Add a descriptive readme file
-Add a screenshot  of the game while is running
-Change a number in the board_spec

**Review by expjazz — APPROVED** [review]
## Status: Project Approved :clap: 

Great job on improving the readme file. It looks great now. Also good job on the tests :ok_hand: 

Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Pull request merged**